internal/pkg/miniprocfs/processes.go

```diff
@@ -164,7 +164,7 @@ func (procs *Processes) readProc(pidString string) (*machine.ProcessInfo, error)
 	var label string
 
 	if err = procs.readFileIntoBuf(path + "attr/current"); err == nil {
-		label = string(bytes.TrimSpace(procs.buf))
+		label = string(bytes.Trim(procs.buf, "\x000\n"))
```

Comment:
Shouldn't it still trim spaces as well?

Reply:
it does, `bytes.Trim` acceps cutstrings `"\x000\n`

Reply:
good point, we can add ` ` as well (idk if it can be there?)